### PR TITLE
GitHub Sync

### PR DESCRIPTION
**Pull request by srawlins**
This is an automatically generated pull request.

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -1,9 +0,0 @@
-# Dependabot configuration file.
```
Comment (devoncarew) [inline .github/dependabot.yml:1]: We want to keep this file :)

File: analysis_options.yaml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 include: package:pedantic/analysis_options.yaml
 analyzer:
-  language:
-    strict-casts: true
+  strong-mode:
```
Comment (devoncarew) [inline analysis_options.yaml:3]: I assume the previous version is the one we want?

**Review by devoncarew — COMMENTED** [review]
It looks like we may want to push the analysis_options.yaml change into google3 first, and update the copybara config to preserve the dependabot file?

File: .github/dependabot.yml (hunk 1 above)
Comment (srawlins) [reply .github/dependabot.yml]: Whoops. Done. :)

File: analysis_options.yaml (hunk 2 above)
Comment (srawlins) [reply analysis_options.yaml]: ah yeah, the woes of having commits on both sides. Fixed.

**Review by devoncarew — APPROVED** [review]
👍

**Comment by srawlins** [comment]
> It looks like we may want to push the analysis_options.yaml change into google3 first, and update the copybara config to preserve the dependabot file?

Yeah, that would be a cleaner way. For now, I just edited the commit.

**Pull request merged**